.circleci/config.yml

```diff
@@ -45,16 +74,23 @@ jobs:
     working_directory: ~/project/android
     docker:
       - image: circleci/android:api-27-node8-alpha
+    environment:
+      TERM: dumb
```

Comment:
What does `TERM: dumb` do?

Reply:
This CircleCI image doesn't set anything for TERM and gradle needs it to be set. I assume Fastlane sets it aswell but now that we're calling gradle directly earlier in the job this was needed. The value dumb is abitrary but recommended.